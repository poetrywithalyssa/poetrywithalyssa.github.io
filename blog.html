<!DOCTYPE html>
<html>
<head>
	<link rel="stylesheet" type="text/css" href="reset.css">
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
	<link rel="stylesheet" type="text/css" href="style.css">
	<link href="https://fonts.googleapis.com/css?family=EB+Garamond|Oswald|PT+Serif" rel="stylesheet">
	<title>BLOG</title>
</head>
<body>
	<header>
		<nav>
    		<div class="nav-wrapper">
      		<a href="index.html" class="brand-logo">Poetry with Alyssa</a>
      		<ul id="nav-mobile" class="right hide-on-med-and-down">
        		<li><a href="index.html">HOME</a></li>
        		<li><a href="about.html">ABOUT</a></li>
        		<li><a href="blog.html">BLOG</a></li>
        		<li><a href="poetry.html">POETRY</a></li>
      		</ul>
    		</div>			
		</nav>
	</header>
	<div class="container">
		<div class="row">
    		<div class="col s12 m8">
      			<div class="card">
        			<div class="card-content white-text">
          			<span class="card-title">WHY POETRY?</span>
          				<p>Hello, interweb denizens. My name’s Alyssa and I’m keen on poetry. Although, I figured you gathered as much from the blog name. However, I suppose you may be questioning why. Why poetry?</p>
        			</div>
        			<div class="card-action">
          				<a href="blog_why_poetry.html">MORE</a>
        			</div>
      			</div>
    		</div>
		</div>
	</div>
</body>
</html>
