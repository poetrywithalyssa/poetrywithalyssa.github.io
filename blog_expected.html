<!DOCTYPE html>
<html>
<head>
	<link rel="stylesheet" type="text/css" href="reset.css">
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
	<link rel="stylesheet" type="text/css" href="style.css">
	<link href="https://fonts.googleapis.com/css?family=EB+Garamond|Oswald|PT+Serif" rel="stylesheet">
	<title>WHAT'S EXPECTED</title>
</head>
<body>
	<header>
		<nav>
    		<div class="nav-wrapper">
      		<a href="index.html" class="brand-logo">Poetry with Alyssa</a>
      		<ul id="nav-mobile" class="right hide-on-med-and-down">
        		<li><a class="nav" href="index.html">HOME</a></li>
        		<li><a class="nav" href="about.html">ABOUT</a></li>
        		<li><a class="nav" href="blog.html">BLOG</a></li>
        		<li><a class="nav" href="poetry.html">POETRY</a></li>
      		</ul>
    		</div>			
		</nav>
	</header>
	<div class="container">
		<div class="row">
			<h1>WHAT'S EXPECTED</h1>
				<p>“I will never study music academically.” Just a few years ago, this phrase would have been absolutely absurd to me. However, when I told my friend this recently, I felt a grin rest comfortably on my face as a wave of relief washed over me. Aside from the music elective I had been placed in this semester, I very swiftly decided to not pursue music for the rest of my collegiate career. </p>
				<div class="col s12 m6">
					<img class="responsive-img" src="IMG_8454.jpg">
					<p class="img-subtext" align="center">Mrs. Murata and I after my final high school concert</p>
				</div>
				<div class="col s12 m6">
					<p>Fairly early on in high school I was pretty firmly set on pursuing a degree in music education. I had been involved with band since fifth grade and the junior school director offered me the TA position when I entered high school. The job primarily entailed sitting in the director's office and correcting student worksheets while occasionally assisting with miscellaneous tasks around the bandroom</p>

					<p>I had a lot of breaks in my schedule freshman year and wasn’t particularly interested in my studies, so when I wasn’t working, I started volunteering at the band room. Mainly, I sat with the trombones to make sure they were paying attention and helped them with slide positions when they were lost. Sometimes, I would work with them in one of the side rooms when they needed help with a section of their concert music or were struggling with a new concept.</p>
				</div>
				<p>I had a lot of breaks in my schedule freshman year and wasn’t particularly interested in my studies, so when I wasn’t working, I started volunteering at the band room. Mainly, I sat with the trombones to make sure they were paying attention and helped them with slide positions when they were lost. Sometimes, I would work with them in one of the side rooms when they needed help with a section of their concert music or were struggling with a new concept.</p>
				<div class="col s12 m6">
					<p>A year and a half went by of my fairly regular assistance when they had their first assembly concert. At this point, mentoring had become one of my favorite pastimes and I looked forward to seeing the kids. And, to a lesser degree, they were excited to see me, too. </p>
				</div>
				<div class="col s12 m6">
					<blockquote>"Mentoring had become one of my favorite pastimes and I looked forward to seeing the kids"</blockquote>
				</div>
				<p>Given the strange nature of my school’s schedule, the high school and junior school schedules were sometimes asynchronous so I had the day off on their concert day. I went to school to help anyway, making sure the kids had their supplies, music, and instruments before leaving the band room to head to the gym.</p>

				<p>Among the ensemble, there was such a nervous fervor. Even though it was more of a music showcase to get the fourth graders interested in signing up for band the following year, it was nonetheless the kids’ second-ever concert. Amongst their wailings that all was ruined and they were going to “suck” I tried to convince them that they would be fine and the concert would go well.</p>

				<p>After helping set up the chairs and making sure everything was squared away with the director, I stood by the gym doors, handing out programs to parents and teachers. With all students and parents seated the concert went underway. It was by no means an absolutely stellar performance, but I was still in awe of how far the band had come. Just three semesters ago they didn’t know how to hold their instruments, and here they were performing four pieces. Knowing I had some minute part it their progress energized and inspired me to go forth on a track of music education.</p>
				<div class="parallax-container">
    				<div class="parallax"><img src=""></div>
    			</div>
				<p>Truly, almost the entirety of that story could have been sampled from any number of my college supplements. But, I always relish recounting it. There’s a fairly attractive wistfulness and naïvity that my sophomore self had that I sometimes wish I can return to. However, the past is in the past, and that distinct moment of energization informed my conduct and pursuit of music for the next two years, which I can only examine as a rueful reflection. </p>

				<div class="col s12 m6">
					<p>Having already been in the top ensemble at school since freshman year, I started taking lessons and slowly started making my way up from last to first chair in my senior year. In addition to concert band, I also added the statewide youth orchestra to my extracurriculars, placing as principal trombonist in the third-highest level.</p>

					<p>Despite my enthusiasm for the youth orchestra and my continued mentoring of the junior school bands, many problems started to arise in the high school’s band program. Without going into specifics, my director, whom everyone loved, was asked to leave on account of the jealousy of another director who inevitably left the school on his own accord. The new director did what he could with my band with extremely polarizing results.</p>
				</div>
				<div class="col s12 m6">
					<img class="responsive-img" src="concert.png">
					<p class="img-subtext" align="center">A section photo taken after the Spring Concert of my sophomore year</p>
				</div>
				<p>Band, which had been my haven for years, swiftly pivoted to becoming an utter bane. This new director’s attention was divided between marching and concert band to which he gave the former priority. The pieces for our Winter concert were laughable and entirely ill-suited for the top ensemble.</p>
				<div class="col s12 m6">
					<blockquote>"I was so hopelessly miserable and started re-contemplating whether or not I should drop band"</blockquote>
				</div>
				<div class="col s12 m6">
					<p>I was so hopelessly miserable and started re-contemplating whether or not I should drop band. By the end of the first semester of my sophomore year, I knew my former director was leaving and had considered dropping band at the end of that year. However, he convinced myself and the majority of the band to stay. For many, he was the only reason why we continued band.</p>
				</div>
				<p>Throughout the course of junior year, my resentment towards band continued to amass and I frequently found myself crying over the state of the program. I knew that I only reacted this way because I cared. But, I never imagined that something I had cherished so much would crumble apart so easily.</p>

				<p>During that year, I began investing myself more into my private studies, even selecting my college audition music in February. Auditions for the next season of youth orchestra were in May, so I spent the entirety of March practicing the excerpts. As a result, I improved significantly, even noting that I had surpassed my skill on piano--an instrument which I had quit lessons for in seventh grade in favor of track--which I never would have imagined.</p>

				<p>I became known as one of the hardcore band kids with the running joke that I lived at the band room. To an extent, there was some truth since I spent many of my breaks in the junior school band room or practicing. I actually rather enjoyed and leaned into the label with several of the underclassmen calling me “band mom.”</p>

				<p>Over the summer, I continued working on my audition piece and started working on my college applications. Per my mom’s advice, I worked on getting all of the written components of my applications out of the way early so I could concentrate on my auditions.</p>
				<div class="col s12 m6">
					<p>At the beginning of the break, I heard back on my youth orchestra auditions and found out that I was in the same ensemble again. Inevitably, I cried. What business did I have pursuing music when I couldn’t even get into the top orchestra; why should I be teaching music when I can’t even play well enough to get into a youth orchestra.</p>
				</div>
				<div class="col s12 m6">
					<blockquote>"What business did I have pursuing music when I couldn’t even get into the top orchestra?</blockquote>
				</div>
				<p>This was devastating to me and I declined the offer. My first year in that level taught me how to be a leader, but beyond that, there wasn’t a particularly reciprocal relationship in terms of what I would get out of the experience of being in that ensemble again. Someone declined their offer in the second-level orchestra so I moved up, but all the same, I was last chair in the second highest orchestra.</p>
				<div class="parallax-container">
    				<div class="parallax"><img src=""></div>
    			</div>
    			<p>As the new school year started, I felt the creep of futurity as all my peers began feverishly working on their college apps. I had most of my written supplements done and knew I would be accepted academically to most of my schools so the pressure was off in that regard. However, the very intense reality of knowing that I may not be accepted to programs because of my audition video was terrifying.</p>

    			<p>Despite this, nothing was quite as overwhelming as the resumption of the joyless barren which was band. Again, the director presented us with an underwhelming assortment of pieces, none of which I practiced. </p>

    			<p>Around this time, a recursive and particularly disturbing idea began to arise every time I played trombone: What if I just injured my wrist? I found myself contemplating this idea frequently. If my wrist was sprained or broken, I wouldn’t be able to hold my trombone. What stopped me from acting upon this impulse was the realization I wouldn’t be able to play piano--I still played periodically for my own amusement--and, as an afterthought, I wouldn’t be able to record my audition video. Rather sorrowfully, I continued practicing, trying to shake off the thought.</p>

    			<p>I made it through the rest of the semester without incident, submitting my auditions with utter relief that the nightly two-hour practice sessions were over. The Winter concert would have passed without much event had it not been for the fact that over half of the seniors were quitting. They were fed up with the director and had not incentive to stay. Personally, I so desperately wished I could join them.</p>
    			<div class="col s12 m6">
    				<blockquote>"I no longer felt the clutch of anxiety seizing me before concerts. Rather, I felt nothing at all"</blockquote>
    			</div>
    			<div class="col s12 m6">
    				<p>Ever since freshman year I had always been a nervous performer, frequently caught between a bought of asthma or anxiety before every performance. However, as the two years went by with the new director, I discovered that I no longer felt the clutch of anxiety seizing me before concerts. Rather, I felt nothing at all. I had cycled from caring too much to utter apathy.</p>
    			</div>
    			<p>The feeling had been developing for a while, but the extremity of it baffled and scared me. Not only did I no longer garner any joy from band, but I also hated playing trombone. It became alarmingly clear that I burned out and, at the time, I didn’t know if I would recover from it.</p>
    			<div class="parallax-container">
    				<div class="parallax"><img src=""></div>
    			</div>
    			<p>As college acceptances started rolling in during the spring semester, I was delighted to find that I was accepted to all the schools I applied to. Of course, there was some deliberation as to which school I should attend, but as the decision date loomed ever closer, I knew what I had to do: I accepted the offer from the one school I didn’t apply to as a music education major.</p>

    			<p>This choice was freeing and I was so relieved. I brought my trombone with me to college--in case I ever wanted to pursue music, I had to keep up my practicing--but didn’t touch it for a month. When I finally did practice it was because that for the first time in a long while, I missed playing.</p>

    			<p>During those two hours, I was utterly euphoric--at last, girl and instrument reunited! But, there was still the unsettling feeling of not knowing precisely why this drift had occurred in the first place. The question of why had troubled me for some time and I ruminated over all possible causes.</p>

    			<p>I knew that attributing my distaste solely to the new director was too reductive since my detachment from music also manifested in youth orchestra where I had no problems with that director. I also briefly wrestled with the idea that perhaps this despise stemmed from my use of it as an emotional crutch during a difficult personal period of my life, but quickly dismissed the idea, knowing that I was just reaching for an explanation. Burnout would be an obvious answer, but surely there would be a recovery period wherein playing for myself would make me enjoy practicing again.</p>

    			<p>And, indeed, there was. I had initially intended pursuing both private lessons and community orchestra in college, but quickly decided to not continue trombone formally. I determined I would get more enjoyment out of the experience if I played for myself. I pursued these infrequent practice sessions happily, even including piano.</p>
    			<div class="col s12 m6">
    				<p>As my first college semester went on and I started getting more work, I found myself devoting more time to piano. One day while practicing, I was struck with a revelation: I had detested trombone since being a trombonist was what was expected of me. </p>
    			</div>
    			<div class="col s12 m6">
    				<blockquote>"I had detested trombone since being a trombonist was what was expected of me"</blockquote>
    			</div>
    			<p>Reflecting on my practice sessions when I was working on college auditions, I realized that the initial deterrent to injuring my wrist was that I wouldn’t be able to play piano. I didn’t think much of it at the time, but this realization absolutely clarified the reasoning.</p>

    			<p>I valued piano because it was a musical pursuit that was solely my own. What I had done by setting a course for music education was box myself in. I became “that trombone girl” with band being my primary identifier. In senior year, I unconsciously tried to work myself out by taking four English classes and working on an independent poetry project. This perhaps slightly modified the general perception of myself, but at the core, I was still “that band kid.”</p>

    			<p>With the familiarity of high school over and college well underway, I can now comfortably say that I enjoy music again. Sure, there’s a nearly zero percent chance that I will ever pursue it academically, but it doesn’t matter--it’s not what I’m expected to do. From now to winter break, I’m hoping to clean up the pieces I’m working on for trombone and piano so I can share my music with friends and family once more.</p>
		</div>
	</div>
</body>
<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
<script type="text/javascript" src="script.js"></script>
</html>